troubleshooting/incrementally_configuring_images.md

```diff
@@ -0,0 +1,297 @@
+# Incrementally configuring images
+
+When IUF or SAT `bootprep` are failing to configure an image, this procedure can be used to incrementally configure an image.
+This avoids re-running the build and configuration steps that have succeeded, which would occur if IUF or SAT `bootprep` were re-run.
+
+The following example assumes the user is creating a `Compute` image, but can be adapted for other image types.
+
```

Comment:
The page is long enough that it may be nice to add a table of contents.

```suggestion

- [Setting up](#setting-up)
- [Generating a partially configured image](#generating-a-partially-configured-image)
- [Restarting SAT and IUF](#restarting-sat-and-iuf)

```